js/libs/form.js

```diff
@@ -1273,7 +1358,21 @@ Fliplet().then(function() {
 
                     var hasChanged = field.value !== value;
 
-                    field.value = value;
+                    if (field._type === 'flMatrix') {
+                      var option = {};
+
+                      _.some(field.rowOptions, function(row) {
```

Comment:
You haven't used `return true` anywhere to short circuit the `_.some` loop.

Reply:
@tonytlwu, done please review.